package.json

```diff
@@ -11,7 +11,6 @@
     "start": "wp server",
     "lint": "phpcs -v wordpress/wp-content/themes/daniel-faria-gallery-headless/. RoboFile.php"
   },
-  "devDependencies": {
-    "babel-plugin-styled-components": "^1.5.1"
-  }
+  "devDependencies": {},
```

Comment:
🔥

Reply:
I don't understand, delete the styled components dev dependency?